static/app/views/starfish/views/appStartup/screenSummary/samples.tsx

```diff
@@ -0,0 +1,112 @@
+import {type ReactNode, useState} from 'react';
+import styled from '@emotion/styled';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import {SegmentedControl} from 'sentry/components/segmentedControl';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {EventSamples} from 'sentry/views/starfish/views/appStartup/screenSummary/eventSamples';
+import {SpanOperationTable} from 'sentry/views/starfish/views/appStartup/screenSummary/spanOperationTable';
+import {SpanOpSelector} from 'sentry/views/starfish/views/appStartup/screenSummary/spanOpSelector';
+import {StartTypeSelector} from 'sentry/views/starfish/views/appStartup/screenSummary/startTypeSelector';
+import {
+  MobileCursors,
+  MobileSortKeys,
+} from 'sentry/views/starfish/views/screens/constants';
+import {DeviceClassSelector} from 'sentry/views/starfish/views/screens/screenLoadSpans/deviceClassSelector';
+
+const EVENT = 'event';
+const SPANS = 'spans';
+
+export function SamplesTables({transactionName}) {
+  const [sampleType, setSampleType] = useState<typeof EVENT | typeof SPANS>(SPANS);
+  const {primaryRelease, secondaryRelease} = useReleaseSelection();
+
+  let content: null | ReactNode = null;
+
+  if (sampleType === EVENT) {
```

Comment:
Can we just call this with `useMemo` for performance?

Reply:
Done! Thanks for the suggestion

https://github.com/getsentry/sentry/pull/64681/commits/b6b5c93229174839b28f42d839f4f267124337c4